notebooks/notebooks/iclr2022/win-rate-vs-board-size.ipynb

```diff
@@ -0,0 +1,311 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    import matplotlib
</pre><p>Nit: PEP8 says imports should be Python standard, then third-party, then first-party. We have a separate block for first-party, but the standard &amp; third-party are currently intermingled. typing &amp; pathlib are Python standard so should come first, then matplotlib/pandas/sgfmill.</p><p> </p><p>We should maybe consider linting for this somehow, although IIRC isort doesn't support this out of the box.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
`nbqa isort win-rate-vs-board-size.ipynb` works